.github/workflows/applicationTesting.yml

```diff
@@ -18,9 +18,9 @@ jobs:
       postgres:
         image: postgres:10.8
         env:
-          POSTGRES_USER: ${{ secrets.POSTGRES_DB_USER }}
-          POSTGRES_PASSWORD: ${{ secrets.POSTGRES_DB_PASS }}
-          POSTGRES_DB: ${{ secrets.POSTGRES_DB_NAME }}
+          POSTGRES_USER: ${{ secrets.WORKFLOW_POSTGRES_DB_USER }}
```

Comment:
for consitency , let keep variable like this... 

env:
          POSTGRES_USER: postgres
          POSTGRES_PASSWORD: postgres
          POSTGRES_DB: github_actions
        ports:

and since it's just for testing no need to keep them in secrets 

If you decide to use variable names use the following : 

`secrets.POSTGRES_DATABASE_TEST`